Update README.md (#594)
embroider-build/ember-auto-import: packages/ember-auto-import/README.md

});
```

> ### I want to import a module for side affects only.

Reviewer: ```suggestion
### I want to import a module for side effects only.
```

Typo

> Some modules, often times polyfills, don't provide values meant for direct import. Instead, the module is meant to provide certain side affects, such as mutating global variables.

Reviewer: There's a blank line after a heading in other paragraphs.

```suggestion

Some modules, often times polyfills, don't provide values meant for direct import. Instead, the module is meant to provide certain side affects, such as mutating global variables.
```

> To import a module for side affects only, you can simply [import the module directly](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#:~:text=Import%20a%20module%20for%20its%20side%20effects%20only).<br>

Reviewer: ```suggestion
To import a module for side affects only, you can simply [import the module directly](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).<br>
```

Removing Google noise from the link.

Reviewer 2: ```suggestion
To import a module for side affects only, you can simply [import the module directly](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#import_a_module_for_its_side_effects_only).<br>
```

Any side affects the module provides will take affect.

> Example:
> The `eventsource` package provides a ready to use [eventsource-polyfill.js](https://github.com/EventSource/eventsource/blob/master/example/eventsource-polyfill.js) module.<br>
> This can be imported like:

Reviewer: ```suggestion
Example: the `eventsource` package provides a ready to use [eventsource-polyfill.js](https://github.com/EventSource/eventsource/blob/master/example/eventsource-polyfill.js) module.

This can be imported like:
```

Cleaning formatting.
